reflex-dom-core/src/Reflex/Dom/Old.hs

```diff
@@ -112,6 +113,7 @@ type MonadWidgetConstraints t m =
   , MonadSample t (Performable m)
   , MonadReflexCreateTrigger t m
   , PostBuild t m
+  , MonadMountStatus t m
```

Comment:
Alright, I think that makes sense; we can keep `MonadWidget` going as a kind of "kitchen sink class" for the time being.